Developer Samples/BuildIrJsRuleApp/BuildIrJsRuleApp/Properties/AssemblyInfo.cs

```diff
@@ -0,0 +1,36 @@
+﻿using System.Reflection;
+using System.Runtime.CompilerServices;
+using System.Runtime.InteropServices;
+
+// General Information about an assembly is controlled through the following
+// set of attributes. Change these attribute values to modify the information
+// associated with an assembly.
+[assembly: AssemblyTitle("BuildIrJsRuleApp")]
+[assembly: AssemblyDescription("Command line utility for requesting compiled irJS Rule Applications from the irDistribution compilation service")]
+[assembly: AssemblyConfiguration("")]
+[assembly: AssemblyCompany("InRule Technologies")]
```

Comment:
Our legal name is InRule Technology, not InRule Technologies.

Reply:
File (and concept of assembly info) was removed during conversion to .net core